docs/topics/vscode-extension-resolving-issues.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-resolving-issues_{context}']
+
+= Resolving issues with the {ProductName} extension
```

Comment:
I think "Resolving migration issues" would be clearer. You don't have to mention the MTA extension in every single title.